package.json

```diff
@@ -93,6 +93,7 @@
     "@nrwl/node": "12.5.9",
     "@nrwl/tao": "12.5.9",
     "@nrwl/workspace": "12.5.9",
+    "@rnx-kit/eslint-plugin": "^0.2.4",
```

Comment:
This should be added as a dependency of eslint-plugin rather than at the root (since eslint-plugin is published)

Reply:
Thanks for the fixes; removed from root.

Reply:
I actually misspoke--what we've started doing is a range under package `dependencies` and a specific version under `devDependencies` at the root to control what's installed.